learning_resources/serializers_test.py

```diff
@@ -266,20 +267,6 @@ def test_learning_resource_serializer(  # noqa: PLR0913
     }
 
 
-def test_learning_resource_serializer_published_runs_only():
-    """Only published runs should be in the serializer"""
-    resource = LearningResourceFactory.create(is_course=True)
-    assert resource.runs.count() == 2
-    assert len(LearningResourceSerializer(resource).data["runs"]) == 2
-    unpublished_run = resource.runs.last()
-    unpublished_run.published = False
-    unpublished_run.save()
-    updated_data = LearningResourceSerializer(resource).data
-    assert len(updated_data["runs"]) == 1
-    assert updated_data["runs"][0]["id"] != unpublished_run.id
```

Comment:
Wondering why this test was removed since it seems like when I re-add this test it does return an unpublished run.

Reply:
Because I removed the code [here](https://github.com/mitodl/mit-open/pull/1303/files#diff-638dcb3aea0f7898675d7c87d8c639d4c572fa2a797f346a62c772db215119d7L525) that was doing the filtering as the filtering is now more appropriately being done in the [query prefetching](https://github.com/mitodl/mit-open/pull/1303/files#diff-3e244d0725c2dbf2dfd444546f36377feb3dcb0bfee9838f50a971237def1e72R289)